interface/wx/grid.h

```diff
@@ -4409,6 +4577,9 @@ class wxGrid : public wxScrolledCanvas
         a grid with a million of columns, we don't want to create an array with
         a million of entries in this function, instead it returns an empty
         array and GetSelectedCols() returns an array containing one element).
+
+        @since 3.1.4 The function can be slow for the big grids, use
```

Comment:
Ha, here is a rare example of a superfluous `@since`. It really shouldn't be used for the functions that existed before 3.1.4. If the function wasn't slow before, you need to write in full words that it became less efficient in 3.1.4 and so the new function should be used instead. But I think it hasn't actually changed, so this could be just removed:
```suggestion
        The function can be slow for the big grids, use
```